http-server/build.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     implementation "io.dropwizard:dropwizard-core:$dropwizardVersion"
     implementation "io.dropwizard:dropwizard-jdbi3:$dropwizardVersion"
     implementation "io.dropwizard:dropwizard-auth:$dropwizardVersion"
+    implementation 'io.github.openfeign:feign-gson:10.2.0'
```

Comment:
What about extracting the version number to a global variable just like the dependencies above are doing it?

Reply:
Since the value is repeated, it makes sense. I wonder if we are adding too much ceremony to our dependencies. Do you recall @RoiEXLab why we switched from `compile` to `implementation`?

Reply:
IIRC this was because gradle updated their build model and made `compile` deprecated.
I believe this has some further implications regarding transitive dependencies, but maybe that was something else I recall from the android build plugin (where there is also `api` as an option)

Reply:
The transitive dependency behavior does seem to be a major difference, it was a bit of a surprise. I'm wondering which is better, explicit dependencies or reduced repetition by inheriting transitive dependencies.